### PR TITLE
Add Github upload at the end of the Sprint

### PR DESCRIPTION
**Pull request by Ana06**
Upload the burndown char and data to Github on end of Sprint.

The following Github account is used for that:

https://github.com/bot-bs

### CODE REVIEW

**Comment by Ana06** [comment]
DO NOT MERGED, I made some changes and I want to double check that everything still works :wink:

File: config/environment.rb [hunk 1]
```diff
@@ -5,3 +5,10 @@
 Rails.application.initialize!
 
 ENV["TROLLOLO_CONFIG_PATH"] = Rails.root.to_s + '/trollolo/.trollolorc'
+
+ENV["GITHUB_USER"] = "bot-bs"
```
Comment (DavidKang) [inline config/environment.rb]: Please change the content by a sample.

File: config/environment.rb [hunk 2]
```diff
@@ -5,3 +5,10 @@
...
+ENV["GITHUB_EMAIL"] = "ammartinez@suse.com"
```
Comment (DavidKang) [inline config/environment.rb]: same here.

**Comment by Ana06** [comment]
This can now be merged! It is tested and the requested changes are made

**Comment by Ana06** [comment]
[(3) P19: Frontpage: Burndown Chart](https://trello.com/c/Gm0FGST1/567-3-p19-frontpage-burndown-chart)

**Pull request merged**